packages/styles/variables.css

```diff
@@ -11,7 +11,9 @@
   --gray-20: #f2f2f2;
   --gray-30: #e0e0e0;
   --gray-40: #cccccc;
+  --gray-50: #b3bfc6;
   --gray-60: #666666;
+  --gray-70: #5d676f;
```

Comment:
This will conflict with the extension and is not a grayscale color, perhaps it should just be local to where its needed?